extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -651,100 +653,83 @@ private function doSearchTags( $searchTag ) {
 	/**
 	 * doMultiRemoveTags
 	 *
-	 * @access private
-	 * @author uberfuzzy@wikia-inc.com
+	 * Working data is stored in object prior to call
+	 *
+	 *   $this->mRemoveTag; has the tag to remove
+	 *   $this->mRemoveTags; has int array of wiki id to remove from
 	 *
-	 * @return text message (use Wikia::*box functions)
+	 * @return string Text message (use Wikia::*box functions)
 	 */
 	private function doMultiRemoveTags( ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 5, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)